openlibrary/templates/account/password/forgot.html

```diff
@@ -4,7 +4,7 @@
 
 <div id="contentHead">
   <h2>$_("Forgot your Archive.org Password?")</h2>
-  <p class="instruct">$_('Forgot your Archive.org password?') <a href="https://archive.org/account/login.forgotpw.php">Click here.</a></p>
+  <p class="instruct">$_('Forgot your Archive.org Password?') <a href="https://archive.org/account/login.forgotpw.php">$_('Click here.')</a></p>
```

Comment:
I originaly changed it to have the exact same form as on line 6 and streamline translators work. It will otherwise result in two different msg in the i18n message template evenso they are the same in meaning and almost the same in form.